mod_reforged/hooks/skills/perks/perk_mastery_hammer.nut

```diff
@@ -0,0 +1,48 @@
+::Reforged.HooksMod.hook("scripts/skills/perks/perk_mastery_hammer", function(q) {
+	q.m.DentArmorSkills <- [
+		"actives.demolish_armor",
+		"actives.crush_armor"
+	];
+	q.m.MinArmorToDent <- 250;
```

Comment:
How about a different name like `ArmorMaxThresholdForDent`?
The `min` in your current name is a useful information which I want to preserve by using `Threshold`. I argue that the `max` part is more important to understand what this variable does.

Reply:
I'm open to changing the name but `ArmorMaxThresholdForDent` is confusing in terms of wheter it is the "MaxThreshold" for "Armor" or is it the "Threshold" for "ArmorMax". Putting an underscore there could clarify it e.g. `ArmorMax_ThresholdForDent` or `Min_ArmorMax_ForDent` or `DentArmorRequiredArmorMax`. Thoughts?

Reply:
Out of everything proposed, MinArmorToDent seems the best combination of lack of ambiguity and cleanliness.